examples/database/postgres.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: database.in-cluster.crossplane.io/v1alpha1
-kind: Postgres
+kind: Operator
```

Comment:
would this kind change to Operator actually work? looks like the spec is still for Postgres type, it doesn't have the fields defined in OperatorParameters

Reply:
Good catch, thanks, Jared! Not sure when I made this change, but it was definitely a typo.